CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 You can find all notable changes to Explorer for Endevor in this document.
 
+## [1.0.0] &ndash; 2022-02-23
+
+### Added
+
+- Telemetry event recording.
+- Added a `[MAP]` folder in the tree view. It is being populated with elements found up the map.
+- Added a button `Show logs` in the notification panel which allows the user to open the extension's output.
+
+### Changed
+
+- Multiple error logging and notification events.
+
+### Fixed
+
+- Fixed an issue in the URI building process. Characters that may potentially break the building process are now being escaped.
+- Upload dialogs now prevent wildcards as input.
```

Comment:
Is that a bug fix or a change?

Reply:
I would vote for a bugfix.

@nickImbirev what do you think?

Reply:
If it's a bug fix, then it should read:

```suggestion
- Fixed an issue that erroneously allowed upload dialogs to use wildcards as input.
```

If not a bug fix, then discard the suggestion